provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -312,11 +312,31 @@ def _get_iface(self, machine_id: str) -> str:
             return 'dummy'
         return ifaces[0]
 
-    # cortx>motr>interface_type
     def _get_iface_type(self, machine_id: str) -> Optional[Protocol]:
-        iface = self.provider.get(
-            'cortx>motr>interface_type',
+        endpoints = self.provider.get(
+            'cortx>hare>hax>endpoints',
             allow_null=True)
+
+        if endpoints is None:
+            return None
+
+        hostname = self.utils.get_hostname(machine_id)
+
+        iface = None
```

Comment:
let's use `proto` instead of `iface`? `proto` seems more relevant.

Reply:
Done